### DISCUSSION TITLE
JavaScript not reloaded when watching

### DISCUSSION THREAD

**Discussion by pmatos** [11ty/eleventy#2621] [Q&A] [answered]
Watching works really well when changing the markdown and saving. However, if my change is in the JS code for a filter, this is not reloaded. Is there a way to reload it in this case?

Thanks,

-- 
Paulo Matos

**Comment by amithm7** [comment, marked as answer]
Add the file / directory to watch target: https://www.11ty.dev/docs/watch-serve/#add-your-own-watch-targets

**Reply by pmatos** [reply]
Thanks.
